tests/slack/slack_payloads.py

```diff
@@ -0,0 +1,229 @@
+users_list_response = {
+    "ok": "True",
+    "members": [
+        {
+            "id": "W012A3CDE",
+            "team_id": "T012AB3C4",
+            "name": "spengler",
+            "deleted": "False",
+            "color": "9f69e7",
+            "real_name": "spengler",
+            "tz": "America/Los_Angeles",
+            "tz_label": "Pacific Daylight Time",
+            "tz_offset": -25200,
+            "profile": {
+                "avatar_hash": "ge3b51ca72de",
+                "status_text": "Print is dead",
+                "status_emoji": ":books:",
+                "real_name": "Egon Spengler",
+                "display_name": "spengler",
+                "real_name_normalized": "Egon Spengler",
+                "display_name_normalized": "spengler",
+                "email": "spengler@ghostbusters.example.com",
+                "image_24": "https://.../avatar/e3b51ca72dee4ef87916ae2b9240df50.jpg",
+                "image_32": "https://.../avatar/e3b51ca72dee4ef87916ae2b9240df50.jpg",
+                "image_48": "https://.../avatar/e3b51ca72dee4ef87916ae2b9240df50.jpg",
+                "image_72": "https://.../avatar/e3b51ca72dee4ef87916ae2b9240df50.jpg",
+                "image_192": "https://.../avatar/e3b51ca72dee4ef87916ae2b9240df50.jpg",
+                "image_512": "https://.../avatar/e3b51ca72dee4ef87916ae2b9240df50.jpg",
+                "team": "T012AB3C4"
+            },
+            "is_admin": "True",
+            "is_owner": "False",
+            "is_primary_owner": "False",
+            "is_restricted": "False",
+            "is_ultra_restricted": "False",
+            "is_bot": "False",
+            "updated": 1502138686,
+            "is_app_user": "False",
+            "has_2fa": "False"
+        },
+        {
+            "id": "U12345678",
+            "team_id": "T0G9PQBBK",
+            "name": "chrisevans",
+            "deleted": "False",
+            "color": "9f69e7",
+            "real_name": "Glinda Southgood",
```

Comment:
😮 @evnsio you never told us about your real name!

Reply:
Dammit 🤦‍♀ .  I thought I'd removed all of mention of Chris in the test data.  Sorry :(